### PR TITLE
DisAllow Trailing/Leading spaces in URLs

### PR DESCRIPTION
**Pull request by VivekVinushanth**
### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 1]
```diff
@@ -907,39 +911,39 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
-                             )
-                             : null
+                                ? (
+                                    <Grid.Row columns={1}>
+                                        <Grid.Column mobile={16} tablet={16} computer={16}>
+                                            <Field
+                                                ref={signatureValidationForArtifactBinding}
+                                                name="signatureValidationForArtifactBinding"
+                                                label=""
+                                                required={false}
```
Comment (thivi) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:921]:
```suggestion
                                                required={ false }
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 2]
```diff
@@ -907,39 +911,39 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
+                                                requiredErrorMessage="this is needed"
```
Comment (thivi) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:922]:
```suggestion
                                                requiredErrorMessage=""
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 3]
```diff
@@ -907,39 +911,39 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
+                                                type="checkbox"
+                                                value={
+                                                    isArtifactBindingEnabled && initialValues?.singleSignOnProfile
+                                                        .enableSignatureValidationForArtifactBinding ?
+                                                        ["enableSignatureValidationForArtifactBinding"] : []}
```
Comment (thivi) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:927]:
```suggestion
                                                    isArtifactBindingEnabled && initialValues?.singleSignOnProfile
                                                        .enableSignatureValidationForArtifactBinding 
                                                            ? ["enableSignatureValidationForArtifactBinding"] 
                                                            : []
                                                }
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 4]
```diff
@@ -907,39 +911,39 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
+                                                children={[
+                                                    {
+                                                        label: t("console:develop.features.applications.forms.inboundSAML" +
+                                                            ".sections.ssoProfile.fields.artifactBinding.label"),
+                                                        value: "enableSignatureValidationForArtifactBinding"
+                                                    }
+                                                ]}
+                                                readOnly={readOnly}
```
Comment (thivi) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:935]:
```suggestion
                                                readOnly={ readOnly }
```
Check other places as well.

**Review by thivi — CHANGES_REQUESTED** [review]

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx (hunk 1 above)
Comment (VivekVinushanth) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Fixed with 9b8bdce8d7b4a57d300b33d30c62e351149c4a51

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx (hunk 4 above)
Comment (VivekVinushanth) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Fixed with 9b8bdce8d7b4a57d300b33d30c62e351149c4a51

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 5]
```diff
@@ -907,39 +911,41 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
-                                            children={[
-                                                {
-                                                    label: t("console:develop.features.applications.forms.inboundSAML" +
-                                                        ".sections.ssoProfile.fields.artifactBinding.label"),
-                                                    value: "enableSignatureValidationForArtifactBinding"
+                                ? (
+                                    <Grid.Row columns={1}>
+                                        <Grid.Column mobile={16} tablet={16} computer={16}>
+                                            <Field
+                                                ref={signatureValidationForArtifactBinding}
```
Comment (thivi) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:918]: Can we leave a space between the curly brace and the prop value? Please check other places as well.

Comment (VivekVinushanth) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Addressed those props in the relevant PR.

**Review by thivi — APPROVED** [review]

**Pull request merged**